sov-modules/sov-app-template/src/lib.rs

```diff
@@ -41,6 +46,22 @@ impl<C: Context, V, RT, H, GenesisConfig> AppTemplate<C, V, RT, H, GenesisConfig
             working_set: None,
         }
     }
+
+    fn revert_and_slash(&mut self, batch_workspace: WorkingSet<C::Storage>) {
+        // Revert all the changes (the sequencer funds are no longer locked)
+        let batch_workspace = batch_workspace.revert();
+
+        // Create a new batch_workspace
+        let mut batch_workspace = batch_workspace.to_revertable();
```

Comment:
Does this one need to be revertable? I don't think we ever revert a slashing, do we?

Reply:
Fixed